content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -175,9 +144,48 @@ const getUsers = await prisma.user.findMany({
 })
 ```
 
+## Filter on a <inlinecode>Json</inlinecode> field (advanced)
+
+You can also filter rows by the data inside a `Json` field. We call this **advanced `Json` filtering**. This functionality is supported by [PostgreSQL](/concepts/database-connectors/postgresql) and [MySQL](/concepts/database-connectors/mysql) only with [different syntaxes for the `path` option](#path-syntax-depending-on-database). 
+
+<Admonition type="warning">
+
+PostgreSQL does not support [filtering on object key values in arrays](#filtering-on-object-key-value-inside-array).
+
+</Admonition>
```

Comment:
Is this still accurate?

Reply:
No idea. It is in the current content, so I left it there.